doc/btp_csip.txt

```diff
@@ -47,21 +47,27 @@ Commands and responses:
 	Opcode 0x04 - Set Coordinator Lock
 
 		Controller Index:	<controller id>
-		Command parameters:     Members count (1 octet)
+		Command parameters:     Address list (varies)
```

Comment:
I'm not sure if sending list with no length is a good idea. Do we assume it's inferred from total command length? I don't think it's a good practice. I'd keep Members count and add the list behind it. Other BTP commands follow that pattern

Reply:
Actually address count is sent along with Address list, just forgot to write it in documentation. 
`addr_cnt = len(addr_list) if addr_list else 0

 data.extend(struct.pack('b', addr_cnt))`